src/main/java/hudson/plugins/git/UserRemoteConfig.java

```diff
@@ -82,10 +83,12 @@ public String toString() {
     @Extension
     public static class DescriptorImpl extends Descriptor<UserRemoteConfig> {
 
+        @SuppressFBWarnings(value="NP_NULL_PARAM_DEREF", justification="pending https://github.com/jenkinsci/credentials-plugin/pull/68")
```

Comment:
Note that this was merged so this comment could be amended to say that the suppression could be deleted as of `credentials` 2.1.9.